Dockerfile

```diff
@@ -1,23 +1,32 @@
-# Build the operator
-FROM golang:1.15.4 AS builder
-WORKDIR /go/src/github.com/kubernetes-sigs/node-feature-discovery-operator
+# Build the manager binary
+FROM golang:1.15 as builder
```

Comment:
nit: should we keep the version pinned to the minor version as it was. Better reproducibility that way